examples/flux-chat/js/utils/ChatMessageUtils.js

```diff
@@ -21,6 +21,18 @@ module.exports = {
       text: rawMessage.text,
       isRead: rawMessage.threadID === currentThreadID
     };
+  },
+
+  getCreatedMessageData: function(text, currentThreadID) {
```

Comment:
I might have named this createMessageData, as it's more generative, not a really a query.